File: Selenium/Activity7.java
package Selenium;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;

public class Activity7 {

	public static void main(String[] args) {
		WebDriver driver = new ChromeDriver();
		driver.get("https://training-support.net/webelements/dynamic-controls");
		System.out.println("Page Title is : " + driver.getTitle());
		WebElement textField = driver.findElement(By.id("textInput"));
		if(textField.isEnabled() == true) {
			System.out.println("field is enabled");
		}
		else {
			System.out.println("field is not enabled");
		}
		driver.findElement(By.id("textInputButton")).click();
		if(textField.isEnabled() == true) {
			System.out.println("Field is enabled");
		}
		else {
			System.out.println("Field is not enabled");
		}
		driver.quit();
	}

}

File: Selenium/Activity7.py
from selenium import webdriver
from selenium.webdriver.common.by import By

with webdriver.Chrome() as driver:
		driver.get("https://training-support.net/webelements/dynamic-controls")
		print("Web age title - " + driver.title)
		textField = driver.find_element(By.ID, "textInput")
		if(textField.is_enabled() == True):
			print("TextField is enabled")
		else:
			print("TextField is not enabled")
		driver.find_element(By.ID,"textInputButton").click()
		if(textField.is_enabled() == True):
			print("Field is enabled")
		else:
			print("Field is not enabled")
		driver.quit()